halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/persistentStorage/S3Validator.java

```diff
@@ -16,15 +16,50 @@
 
 package com.netflix.spinnaker.halyard.config.validate.v1.persistentStorage;
 
+import com.amazonaws.SdkClientException;
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.Bucket;
+import com.netflix.spinnaker.front50.config.S3Config;
+import com.netflix.spinnaker.front50.config.S3Properties;
 import com.netflix.spinnaker.halyard.config.model.v1.node.Validator;
 import com.netflix.spinnaker.halyard.config.model.v1.persistentStorage.S3PersistentStore;
 import com.netflix.spinnaker.halyard.config.problem.v1.ConfigProblemSetBuilder;
+import com.netflix.spinnaker.halyard.core.problem.v1.Problem;
+import com.netflix.spinnaker.halyard.core.tasks.v1.DaemonTaskHandler;
 import org.springframework.stereotype.Component;
 
+import java.util.Optional;
+
 @Component
 public class S3Validator extends Validator<S3PersistentStore> {
   @Override
   public void validate(ConfigProblemSetBuilder ps, S3PersistentStore n) {
-    // TODO(lwander) find someone to handle https://github.com/spinnaker/halyard/issues/116
+    DaemonTaskHandler.message(String.format("Validating %s with %s", n.getNodeName(), getClass().getSimpleName()));
+    AWSCredentialsProvider credentialsProvider = DefaultAWSCredentialsProviderChain.getInstance();
```

Comment:
Halyard already has a dependency on front50, but it is pinned to version 1.72.0 in spinnaker-dependencies. Should I update the dependency there, or should I add an explicit dependency on a newer front50 version? In 1.72.0 your implementation of `ensureBucketExists()` doesn't exist yet.